scripts/env-prep

```diff
@@ -4,11 +4,24 @@ if [ -n "${SDEBUG:-}" ] ; then
   set -x
 fi
 
+# Override default logging namespace
+# Try 'openshift-logging' or 'logging'
 if [ -f ".logging-ns" ] ; then
   LOGGING_NS=$(cat .logging-ns)
 fi
 
+# Override default elasticsearch component
+# Try 'elasticsearch' or 'es'
+if [ -f ".es-component" ] ; then
+  ES_COMPONENT=$(cat .es-component)
```

Comment:
I'm not sure it makes sense to do this given 3.x and 4.x deployments are different and there is some drift.